contracts/libraries/VaultLifecycle.sol

```diff
@@ -5,7 +5,7 @@ import {SafeMath} from "@openzeppelin/contracts/utils/math/SafeMath.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {Vault} from "./Vault.sol";
 import {ShareMath} from "./ShareMath.sol";
-import {IStrikeSelection} from "../interfaces/IRibbon.sol";
+import {IDeltaStrikeSelection} from "../interfaces/IRibbon.sol";
```

Comment:
We can stick to IStrikeSelection https://github.com/ribbon-finance/ribbon-v2/pull/187/files#r770235566